test/test_ui.py

```diff
@@ -865,6 +865,9 @@ def setUp(self):
 
         # Custom BEETSDIR
         self.beetsdir = os.path.join(self.temp_dir, b"beetsdir")
+        self.cli_config_path = os.path.join(
+            os.fsdecode(self.temp_dir), "config.yaml"
+        )
```

Comment:
```suggestion
        self.cli_config_path = os.path.join(self.temp_dir, b"config.yaml")
```

Let's stick to the convention of storing paths internally as `bytes` for now, until a change to a `pathlib` future is actually underway.

Reply:
I don't think this is a good idea, since `run_command` does not support `bytes`.

This is precisely why I removed `_convert_args` from test helpers which previously decoded `bytes`-based args to `str` for some tests:

![image](https://github.com/user-attachments/assets/6a1f046b-8ea4-472c-8146-5b832e713722)

There's no need for back-and-forth encoding and decoding when we just need to use strings.

Reply:
Makes sense. I was thinking too much in terms of implementation rather than testing code: In `/beets/...` I'd argue that my point on sticking to `bytes` for now stands, but for testing code as it is here, that might not be the best choice.